lib/galaxy/datatypes/text.py

```diff
@@ -52,11 +54,7 @@ def sniff_prefix(self, file_prefix):
         >>> Html().sniff( fname )
         True
         """
-        headers = iter_headers(file_prefix, None)
-        for hdr in headers:
-            if hdr and hdr[0].lower().find('<html>') >= 0:
-                return True
-        return False
+        return check_html(file_prefix.filename)
```

Comment:
While reusing code seems useful, I'm not sure here. Searching for `html` tags seems more general than what check_html does.